org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/JDTLanguageServer.java

```diff
@@ -448,6 +449,16 @@ public void projectConfigurationUpdate(TextDocumentIdentifier param) {
 		handler.updateConfiguration(param);
 	}
 
+	/* (non-Javadoc)
+	 * @see org.eclipse.jdt.ls.core.internal.JavaProtocolExtensions#buildWorkspace(java.lang.String)
+	 */
+	@Override
+	public CompletableFuture<BuildWorkspaceResult> buildWorkspace(String type) {
```

Comment:
What are the expected values for type? At the moment it looks like it is not used, we need to clarify its purpose.

Reply:
I think it could be removed